File: backend/CMS_Test/CMS_Test/Models/Stock.cs
using System;
using System.Collections.Generic;

namespace CMS_Test.Models
{
    public class Stock
    {
        public int ID { get; set; }
        public String RegNumber { get; set; }
        public String Make { get; set; }
        public String Model { get; set; }
        public int ModelYear { get; set; }
        public int KMS { get; set; }
        public String Colour { get; set; }
        public String VIN { get; set; }
        public int RetailPrice { get; set; }
        public int CostPrice { get; set; }
        public List<Accessory> Accessories { get; set; }
        public List<Image> Images { get; set; }
        public DateTime DTCreated { get; set; }
        public DateTime DTUpdated { get; set; }
    }
}

File: frontend/cmstest/src/app/models/Stock.ts
import Image from './Image';
import Accessory from './Accessory';

export default class Stock {
    public ID:number;
    public RegNumber:string;
    public Make:string;
    public Model:string;
    public ModelYear:string;
    public KMS:number;
    public Colour:string;
    public VIN:string;
    public RetailPrice:number;
    public CostPrice:number;
    public Accessories:Accessory[];
    public Images:Image[];
    public DTCreated:Date;
    public DTUpdated:Date;
}
